packages/react/src/globals.ts

```
@@ -25,7 +25,7 @@ declare module 'react' {
     }
   }
 }
-
+// biome-ignore lint/suspicious/noTsIgnore: not needed
```

Biome (2.1.2)

[error] Failed to parse category "lint/suspicious/noTsIgnore". Example of suppression: // biome-ignore lint: reason

Biome can't recognize the category, usually it's because it doesn't match the expected ones.
Example of suppression: // biome-ignore lint: reason

(suppressions/parse)